app/classifier/tasks/multiple.jsx

```diff
@@ -0,0 +1,202 @@
+import { Markdown } from 'markdownz';
+import React from 'react';
+import GenericTask from './generic';
+import GenericTaskEditor from './generic-editor';
+
+const NOOP = Function.prototype;
+
+class MultipleChoiceSummary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.expand = this.expand.bind(this);
+    this.collapse = this.collapse.bind(this);
+    this.state = {
+      expanded: this.props.expanded
+    };
+  }
+
+  expand() {
+    this.setState({ expanded: true });
+  }
+
+  collapse() {
+    this.setState({ expanded: false });
+  }
+
+  render() {
+    let toggleButton = <button type="button" className="toggle-more" onClick={this.expand}>More</button>;
+    let answers = <div className="answer">No answer</div>;
+    if (!this.state.expanded && this.props.annotation.value.length > 0) {
+      answers = [];
+      for (const index of this.props.annotation.value) {
```

Comment:
You could use `Array.map` here to map the values array directly to the answers array.